.travis.yml

```diff
@@ -104,6 +104,7 @@ install:
         python -c 'import mne; mne.datasets.testing.data_path(verbose=True)';
         if [ "${DEPS}" == "full" ]; then
           export FREESURFER_HOME=$(python -c 'import mne; print(mne.datasets.testing.data_path())');
+          export MNE_SKIP_FLASH_CALL=1;
```

Comment:
Is this done for speed, or because Travis and AppVeyor don't have the right command-line utilities to do it?